implementations/c/test/ockam/transport/server/test_data_client.txt

```diff
@@ -0,0 +1,358 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
about this and other test data files:

1. could we replace them with output from `/dev/urandom` written to an *uncommitted temp files*?
2. if there is a good reason we can identify to commit these files then I think it would be better to use data that looks random (lorem impsum or random hex) .. the current log output may confuse casual readers / beginners grep-ing in the code base.

Reply:
Agree about using a file with generic content. But, I'm not sure how to use /dev/urandom as a test case, when the client and server might be on different machines. For now, I'll change the content as you suggest.

Reply:
I created a test file with "lorem ipsum"...etc. For now it lives in the source directory. At some point we may want to have a strategy for where such files live.